README.md

```diff
@@ -18,8 +18,9 @@ WARNING: Currently uses plain-text private keys for transactions and is otherwis
 We use glide for dependency management.  The prefered way of compiling from source is the following:
 
 ```
-go get -d github.com/tendermint/basecoin/cmd/basecoin
+go get -u github.com/tendermint/basecoin
```

Comment:
Why wouldn't it work? 

I think the ideal thing, when everything is master, is to just `go get -u github.com/tendermint/basecoin/cmd/basecoin` with nothing else. While we're on develop, we have to do some extra crapola